commit 41eed87c79ebb56e85277198423f168362583fb7
Author: Andy Green <andy@warmcat.com>
Date:   Fri Dec 8 07:48:09 2023 +0000

    cmake: upgrade everyone to 3.5 minimum version

Origin: backport, https://github.com/warmcat/libwebsockets/commit/41eed87c79ebb56e85277198423f168362583fb7

This patch has been adjusted to apply to the version we ship.  It's
very different from the upstream version.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 57c35232..eda93224 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,7 +22,7 @@
 # IN THE SOFTWARE.
 #
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5)
 include(CheckFunctionExists)
 include(CheckSymbolExists)
 include(CheckIncludeFile)
